sklearn/linear_model/_cd_fast.pyx

```diff
@@ -914,6 +914,48 @@ cdef (floating, floating) gap_enet_gram(
     return gap, dual_norm_XtA
 
 
+cdef inline bint screen_features_enet_gram(
```

Comment:
I am a bit surprised that returning `bint` does not lead to a Cython compiler error. But maybe they are both aliases for the same type for the platforms we support.

```suggestion
cdef inline uint32_t screen_features_enet_gram(
```